internal/core/graphics.rs

```diff
@@ -224,6 +224,17 @@ impl From<RequestedOpenGLVersion> for RequestedGraphicsAPI {
     }
 }
 
+/// Private API exposed to just the renderers to create GraphicsAPI instance with
+/// non-exhaustive enum variant.
+#[cfg(feature = "unstable-wgpu-24")]
+pub fn create_graphics_api_wgpu_24(
```

Comment:
Why not put that in the wgpu_24 module? Is that module fully public?

Reply:
Yeah, it's fully public :-/